qemu/tests/transfer_file_over_ipv6.py

```diff
@@ -100,42 +119,50 @@ def get_file_md5sum(file_name, session, timeout):
                                  timeout=file_trans_timeout,
                                  interface=host_ifname)
-            dst_md5 = get_file_md5sum(guest_path, sessions[vm],
-                                      timeout=file_md5_check_timeout)
+            dst_md5 = get_file_md5sum(
+                guest_path, sessions[vm], timeout=file_md5_check_timeout)
+            error_context.context("md5 value of data in %s: %s" % (vm.name, dst_md5),
+                                  logging.info)
             if dst_md5 != src_md5:
                 test.fail("File changed after transfer host -> %s" % vm.name)
 
-        for vm_src in addresses:
-            for vm_dst in addresses:
-                if vm_src != vm_dst:
-                    error_context.context("Transferring data from %s to %s" %
-                                          (vm_src.name, vm_dst.name),
-                                          logging.info)
-                    remote.scp_between_remotes(addresses[vm_src],
-                                               addresses[vm_dst],
-                                               port, password, password,
-                                               username, username,
-                                               guest_path, dest_path,
-                                               timeout=file_trans_timeout,
-                                               src_inter=host_ifname,
-                                               dst_inter=inet_name[vm_src])
-                    dst_md5 = get_file_md5sum(dest_path, sessions[vm_dst],
-                                              timeout=file_md5_check_timeout)
-                    if dst_md5 != src_md5:
-                        test.fail("File changed transfer %s -> %s"
-                                  % (vm_src.name, vm_dst.name))
+        if params.get("os_type") == "linux":
+            for vm_src in addresses:
+                for vm_dst in addresses:
+                    if vm_src != vm_dst:
+                        error_context.context("Transferring data from %s to %s" %
+                                              (vm_src.name, vm_dst.name),
+                                              logging.info)
+                        remote.scp_between_remotes(addresses[vm_src],
+                                                   addresses[vm_dst],
+                                                   port, password, password,
+                                                   username, username,
+                                                   guest_path, dest_path,
+                                                   timeout=file_trans_timeout,
+                                                   src_inter=host_ifname,
+                                                   dst_inter=inet_name[vm_src])
+                        dst_md5 = get_file_md5sum(
+                            dest_path, sessions[vm_dst], timeout=file_md5_check_timeout)
```

Comment:
pay attention to the indent. 
the original one is acceptable.

Reply:
@xiywang Updated.